### PR TITLE
Enable test_meta_xpu

### PR DESCRIPTION
**Pull request by yuchengliu1**
enable test_meta_xpu.
triage in https://github.com/intel/torch-xpu-ops/issues/774

### CODE REVIEW

File: test/xpu/xpu_test_utils.py [hunk 1]
```diff
@@ -255,6 +255,9 @@
     ("logcumsumexp", "test_out_warning"),
     ("_refs.mul", "test_python_ref"),
     ("_refs.mul", "test_python_ref_torch_fallback"),
+    ("narrow_copy","test_meta_outplace"),
+    ("narrow_copy","test_dispatch_meta_outplace"),
+    ("narrow_copy","test_dispatch_symbolic_meta_outplace"),
```
Comment (fengyuan14) [inline test/xpu/xpu_test_utils.py:260]: Does CPU pass on these three?

Comment (yuchengliu1) [reply test/xpu/xpu_test_utils.py:260]: yes, these cases pass on cpu

File: test/xpu/run_test_with_skip.py [hunk 2]
```diff
@@ -5,10 +5,16 @@
...
+fail_test = []
 
 for key in skip_dict:
     skip_list = skip_dict[key]
-    res += launch_test(key, skip_list)
+    fail = launch_test(key, skip_list)
+    res += fail
+    if fail:
+        fail_test.append(key)
+if fail_test:
```
Comment (daisyden) [inline test/xpu/run_test_with_skip.py:16]: What is the fail_test list for?

Comment (yuchengliu1) [reply test/xpu/run_test_with_skip.py:16]: A summary of all UTs in CI. Easy to find failed UT.

**Review by daisyden — APPROVED** [review]

**Pull request merged**